explorer/interpreter/action.h

```diff
@@ -80,13 +76,17 @@ class RuntimeScope {
   // Return scope state
   // Normal     = Scope is not bind at the top of a destructor call
   // Destructor = Scope is bind to a destructor call and was not cleaned up,
+  // Method     = Scope is bind at the top of a method
```

Comment:
Why can't a method be treated as equivalent to other scopes?

Reply:
It is iterated through all LValues of a scope. In the case of a method, the me-value is in the scope and therefore the destructor of the object would be called after the method call.